rbi/sord.rbi

```diff
@@ -1,4 +1,3 @@
-# typed: strong
```

Comment:
?

Reply:
Ah; this is a Parlour bug. I'll fix it manually in this case and then update Parlour shortly